lib/rgeo/coord_sys/cs/entities.rb

```diff
@@ -939,6 +939,15 @@ def get_axis(dimension)
         def get_units(dimension)
           nil
         end
+
+        class << self
+          def create(defn, *optional)
+            # TODO: should this raise?
```

Comment:
I guess raising is the consistent behaviour nowadays, what bothers you?

Reply:
Yeah I think the issue with this and the above comment is that if you're not using proj and just want to use a factory, then it would raise on instantiation if you specify an srid.

ex. RGeo::Geos.factory(srid: 4326) would raise with an `InvalidCoordinateSystem` error or something. Obviously not ideal, so maybe we should raise here, but rescue and assign null in the factory creation logic.

Perhaps we could actually give it a coordinate system but it raises once you try to transform because there's not valid transformation? That might actually be the best bet.